WireDomain/Sources/WireDomain/Repositories/User/UserRepository.swift

```diff
@@ -94,6 +94,15 @@ public protocol UserRepositoryProtocol {
 
     func disableUserLegalHold() async throws
 
+    /// Deletes a user property.
+    ///
+    /// - parameters:
+    ///     - key: The user property key to delete.
+
+    func deleteUserProperty(
+        withKey key: String
```

Comment:
What do you think about using `UserProperty.Key` here instead of `String`?

Reply:
fixed in https://github.com/wireapp/wire-ios/pull/1975/commits/75bc09956e23fd2e48edcdcd74e20178eeb5eb05